src/WebJobs.Script/Diagnostics/FileWriter.cs

```diff
@@ -39,8 +39,8 @@ public FileWriter(string logFilePath)
             // start a timer to flush accumulated logs in batches
             _flushTimer = new Timer
             {
-                AutoReset = true,
-                Interval = LogFlushIntervalMs
+                AutoReset = false,
```

Comment:
Analysis of the dump file also showed many outstanding *concurrent* flush operations coming from timer invocations. I.e. when under load, file contention might result in many timer invocations backing up waiting of file operations, with many invocations outstanding concurrently.

Instead we want there to be at most 1 file flush per interval.